RELEASE.md

```diff
@@ -10,7 +10,9 @@ Our goal is to provide a new minor release every 4 weeks. This is a new process
 |----------------|--------------------------------------------|---------------------------------------------|
 | v0.1.0         | 2019-07-31                                 | Chris Marchbanks (GitHub: @csmarchbanks)    |
 | v0.2.0         | 2019-08-28                                 | Goutham Veeramachaneni (Github: @gouthamve) |
-| v0.3.0         | 2019-10-09                                 | **searching for volunteer**                 |
+| v0.3.0         | 2019-10-09                                 | Bryan Boreham (@bboreham)                   |
+| v0.4.0         | 2019-11-13                                 | Tom Wilkie (@tomwilkie)                     |
+| v0.5.0         | 2019-12-11                                 | **searching for volunteer**                 |
```

Comment:
Adjusted for the previous suggestion.
```suggestion
| v0.5.0         | 2019-12-04                                 | **searching for volunteer**                 |
```

Reply:
I believe Tom is at AWS Reinvent that week.

Reply:
Oops, ignore me - Tom is the month before.